v22.1/cockroach-cert.md

```diff
@@ -3,121 +3,44 @@ title: cockroach cert
 
-To learn more, explore:
+- Nodes
+- Clients
+- DB Console (optional)
 
-- [Transport Layer Security (TLS) and Public Key Infrastructure (PKI)](security-reference/transport-layer-security.html)
-- [Using the CockroachDB CLI to provision a development cluster](manage-certs-cli.html).
-- [Managing CockroachDB security certificates with HashiCorp Vault](manage-certs-vault.html).
+To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, use [`openssl` commands](https://wiki.openssl.org/index.php/), or use a [custom CA](create-security-certificates-custom-ca.html) (for example, a public CA or your organizational CA).
 
-{{site.data.alerts.callout_info}}
-The ability to rapidly and locally generate private key/public certificate pairs is handy for development, but careful management of security certificates is an essential component of cluster security, and performing these tasks with a cloud-native tool such as Google Cloud Platform's Certificate Authority Service (CAS) offers many security advantages.
-
-{{site.data.alerts.end}}
-
-The CLI offers the following functionality:
-
-- Generate a root certificate authority (CA) certificate for your cluster, and use it to sign public certificates.
-- Generate key pair for use by a CockroachDB node.
-- Generate a key pair for use by a TLS-authenticated CockroachDB client.
-
-## Subcommands
-
-### `create-ca`
-
-Create a certificate authorit a self-signed certificate authority (CA) key pair (private key and public certificate), which you'll use to create and authenticate certificates for your entire cluster.
-
-{% include_cached copy-clipboard.html %}
-~~~ shell
-cockroach cert create-ca \
- --certs-dir=[path-to-certs-directory] \
- --ca-key=[path-to-ca-key]
-~~~
+{% include filter-tabs.md %}
 
-### `create-node`
+{{site.data.alerts.callout_success}}For details about when and how to change security certificates without restarting nodes, see <a href="rotate-certificates.html">Rotate Security Certificates</a>.{{site.data.alerts.end}}
 
-Create a certificate and key for a specific node in the cluster. You specify all addresses at which the node can be reached and pass appropriate flags.
+## How security certificates work
 
-Create a node certificate and key:
-
-{% include_cached copy-clipboard.html %}
-~~~ shell
-cockroach cert create-node \
-  [node-hostname] \
-  [node-other-hostname] \
-  [node-yet-another-hostname] \
-  [hostname-in-wildcard-notation] \
-  --certs-dir=[path-to-certs-directory] \
-  --ca-key=[path-to-ca-key]
-~~~
-
-### `create-client`
+1. Using the `cockroach cert` command, you create a CA certificate and key and then node and client certificates that are signed by the CA certificate. Since you need access to a copy of the CA certificate and key to create node and client certs, it's best to create everything in one place.
```

Comment:
In some places you say "certs" and others "certificates". It's probably best to use the whole word everywhere.